src/app/Main/Footer/Footer-Layout.component.tsx

```diff
@@ -17,11 +17,13 @@ const footerStyle = css`
     border-top: 2px solid ${primaryColor};
 `;
 
-export function FooterLayoutComponent(): JSX.Element {
+export function FooterLayoutComponent(props: {
```

Comment:
Reminder: This is no layout component for now, it would be if it would look like:
```
export function FooterLayoutComponent(serverInfoComponent: JSX.Element, linkListComponent: JSX.Element`) {
  return (
    <footer>
      <serverInfoComponent>
      <linkListComponent>
    </footer>
  );
}
```
And in the FooterContainer:
```
// sorry don't now how it would look in react, so pseudo code
return <FooterLayoutComponent 
  serverInfoComponent = <LastUpdateComponent>
  linkListComponent = <FooterLinkListComponent supportMail={props.supportMail}>
  />
```

Reply:
Done. Commit "Refactor FooterLayout"